articles/fuzzy-logic-operations/index.md

```diff
@@ -0,0 +1,138 @@
+Fuzzy logic is an approach to solving problems that take into consideration the degrees of truth. Fuzzy Systems as a subject developed to model the uncertainty and vagueness present in the human thought process. 
+
+In this article, we will understand the following concepts.
+
+### Prerequisites
+
+This article on fuzzy logic is an excellent introduction to the basics of crisp and fuzzy sets. Moreover, it introduces the representation of fuzzy sets and their applications. 
+
+### Table of contents
+
+- Degree of membership function
+- Types of membership functions
+- Operations in fuzzy systems
+- Types of operations
+
+### Degree of membership function
+
+A fuzzy set is denoted as a tuple that includes the element and a value indicating its degree of membership to the universal set. Mathematically, it is represented as:
+
+$$ F= (S,\;\mu(S))\; ;\;S\, \epsilon\, x\; \&\; \mu(S)\:=\:Degree\, of\, S $$
+ 
+The above equation can be viewed as a collection of ordered pairs. The inclusion of the element $S \epsilon\; x$ into the fuzzy set $F$ is fuzzy.
+
+#### Example
+
+Consider the example of temperature control. We will explore the various membership functions using this example. The task to control the room temperature can be modeled as a fuzzy problem. Let the ranges of temperature be given as follows:
```

Comment:
A membership function should also be defined

Reply:
My bad. The prerequisite article to this defines the terminology in detail. I have added the link.